R/baggr.R

```diff
@@ -11,7 +11,7 @@
 #'
 #' @param data data frame with summary or individual level data to meta-analyse;
 #'             see Details section for how to format your data
-#' @param model if \code{NULL}, detected automatically from input data
+#' @param label if \code{NULL}, detected automatically from input data
```

Comment:
This is wrong. Did you mean to replace `effect` with `label`? `model` is something wrong

Reply:
Yes, this was a mistake. Fixed this.